collections/harmonized-landsat-sentinel.yaml

```diff
@@ -0,0 +1,526 @@
+Documentation: "[here](https://docs.sentinel-hub.com/api/latest/data/hls/)"
+Resolution: 30m
+GeographicalCoverage: Globally
+TemporalAvailability: April 2013 - ongoing
+UpdateFrequency: 2-3 days
+BandInformation: "Information about [bands and data](https://docs.sentinel-hub.com/api/latest/data/hls/#available-bands-and-data)"
+Contact: https://forum.sentinel-hub.com/c/euro-data-cube/25
+Provider: "[Land Processes Distributed Active Archive Center (LP DAAC)](https://lpdaac.usgs.gov/)"
+ManagedBy: "[Sentinel Hub](https://www.sentinel-hub.com/)"
+Tags:
+  - xcube
+  - sentinel hub
+  - raster
+  - systematic
+  - satellite imagery
+  - multi spectral imagery
+  - agriculture
+  - natural resource
+  - disaster response
+  - vegetation monitoring
+  - LULC mapping
+  - LULC change
+  - open data
+  - race challenges
+  - copernicus
+  - sentinel
+  - landsat
+License: |
+    "[License](https://www.usgs.gov/centers/eros/data-citation?qt-science_support_page_related_con=0#qt-science_support_page_related_con)"
+    "[License](https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l1c/#attribution-and-use)"
+LicenseType: proprietary
+LicenseUrl: 
+    https://www.usgs.gov/centers/eros/data-citation?qt-science_support_page_related_con=0#qt-science_support_page_related_con
+    https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l1c/#attribution-and-use
+Resources:
+  - Group: Sentinel Hub Resources
+    EndPoint: https://services-uswest2.sentinel-hub.com/api
+    Name: Sentinel Hub
+    Role: processor
+    Type: hls
+    Notes: Globally since April 2013
+    Primary: true
+CustomScripts:
+  Title:
+  URL:
+WMTS:
+  - href:
+    layer_id:
+    dimensions:
+      warnings: true
+Configurations:
+  - href:
+    rel:
+    type:
+    title:
+  - href:
+    rel:
+    type:
+    title:
+    sentinelhub:layer_name:
+    sentinelhub:mosaicking_order:
+    sentinelhub:upsampling:
+Extent:
+  spatial:
+    bbox:
+      -
+        - -180
+        - -80
+        - 180
+        - 84
+  temporal:
+    interval:
+      -
+        - '2013-04-01T00:00:00Z'
+        - null
+CubeDimensions:
+  x:
+    type: spatial
+    axis: x
+    extent:
+      - -180
+      - 180
+    reference_system: 
+      $schema: https://proj.org/schemas/v0.4/projjson.schema.json
+      type: GeodeticCRS
+      name: AUTO 42001 (Universal Transverse Mercator)
+      datum:
+        type: GeodeticReferenceFrame
+        name: World Geodetic System 1984
+        ellipsoid:
+          name: WGS 84
+          semi_major_axis: 6378137
+          inverse_flattening: 298.257223563
+      coordinate_system:
+        subtype: ellipsoidal
+        axis:
+          - name: Geodetic latitude
+            abbreviation: Lat
+            direction: north
+            unit: degree
+          - name: Geodetic longitude
+            abbreviation: Lon
+            direction: east
+            unit: degree
+      area: World
+      bbox:
+        south_latitude: -80
```

Comment:
Where did you get this numbers from? In our catalog api (https://services-uswest2.sentinel-hub.com/api/v1/catalog/1.0.0/collections/hls) we return -85 and 85,

Reply:
I thought this was supposed to be adjusted with the [HLS Tiling System](https://hls.gsfc.nasa.gov/products-description/tiling-system/). Changed it now to be adjusted with the catalog returns.

Reply:
It's probably correct what you had there before, but let's be in sync with our catalog (in the end the only thing that happens if someone searches data far north or south he might get nothing returned.